### PR TITLE
Updated parameters in Innodb Cluster Job

### PR DESCRIPTION
**Pull request by kaushikpuneet07**
Below parameters are updated for Oracle -Linux-8:
* Image
* ssh_User
* instance_type
* Vagrant Box

### CODE REVIEW

File: molecule/ps-innodb-cluster/Jenkinsfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ pipeline {
     string(name: 'UPSTREAM_VERSION', defaultValue: '8.0.26', description: 'Upstream MySQL version')
     string(name: 'PS_VERSION', defaultValue: '16', description: 'Percona part of version')
     string(name: 'PS_REVISION', defaultValue: '3d64165', description: 'Short git hash for release')
-    choice(name: 'TEST_DIST', choices: ['bento/ubuntu-20.04','bento/ubuntu-18.04','bento/debian-10','bento/debian-9','bento/centos-8','bento/centos-7','generic/oracle8'], description: 'On which image to run test.')
+    choice(name: 'TEST_DIST', choices: ['bento/ubuntu-20.04','bento/ubuntu-18.04','bento/debian-10','bento/debian-9','bento/centos-8','bento/centos-7','bento/oracle-8'], description: 'On which image to run test.')
```
Comment (eleo007) [inline molecule/ps-innodb-cluster/Jenkinsfile:9]: Why do we leave centos-8?

**Review by eleo007 — COMMENTED** [review]
Lets use the same naming - either oracle-linux-8 or oracle-8 in the  TEST_DIST and folder names and scenario name. I would stick to oracle-8 as it is shorter one.

File: molecule/ps-innodb-cluster/Jenkinsfile (hunk 1 above)
Comment (kaushikpuneet07) [reply molecule/ps-innodb-cluster/Jenkinsfile:9]: I will remove Deb-9 and Centos-8 when i create a PR to add Oracle Linux 9 and Ubuntu Jammy

Comment (kaushikpuneet07) [reply molecule/ps-innodb-cluster/Jenkinsfile]: Removed Deb-9 and Centos-8

Comment (kaushikpuneet07) [reply molecule/ps-innodb-cluster/Jenkinsfile]: Names are now in sync also

**Review by eleo007 — APPROVED** [review]

**Pull request merged**